lib/Github/Api/Organization/Actions/SelfHostedRunners.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace Github\Api\Organization\Actions;
+
+use Github\Api\AbstractApi;
+
+class SelfHostedRunners extends AbstractApi
+{
+    /**
+     * @link https://docs.github.com/en/rest/actions/self-hosted-runners?apiVersion=2022-11-28#list-self-hosted-runners-for-an-organization
+     *
+     * @param string $organization
+     * @param string $type
+     * @param int    $page
+     *
+     * @return array|string
+     */
+    public function all(string $organization, string $type = 'all', int $page = 1)
```

Comment:
Don't add the page parameter here, if you want to paginate over all results the `ResultPager` class should be used.

Also just add a parameters array to pass the type to the call, this makes the method future proof if github decides to add more options

Reply:
Thanks, replaced with parameters array